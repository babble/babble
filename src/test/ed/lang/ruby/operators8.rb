
p = /a/

text = "ceaf";
a = []
a << "abc" if text =~ p
puts text
puts a.length
puts a[0]

text = "cef";
a = []
a << "abc" if text =~ p
puts text
puts a.length

5.times do
  puts "hi"
end
