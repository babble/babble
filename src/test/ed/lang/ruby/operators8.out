ceaf
1
abc
cef
0
